_posts/2019-07-23-extracting-fsspec-from-dask.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: Extracting fsspec from Dask
+author: Martin Durant
+tags: [IO]
+draft: true
+theme: twitter
+---
+{% include JB/setup %}
+
+
+Summary
+-------
+
+Over the past few years, Dask's IO capability has grown gradually and organically, to
+include a number of file-formats, and the ability to access data seamlessly on various
+remote/cloud data systems. This has been achieved through a number of sister packages
+for viewing cloud resources as file-systems, and dedicated code in the `bytes`
+subpackage within Dask. For the sake of consolidating the behaviours of the
```

Comment:
```suggestion
module within Dask. For the sake of consolidating the behaviours of the
```

Reply:
I propose that while consolidating behavior may have been our internal goal, use by the broader community might speak more to readers that are not Dask developers.  

You might point out how other projects, like Pandas and Xarray/Zarr use projects like`s3fs` and `gcsfs` today without Dask.  This message of "Dask's data access tooling is now generally available without Dask" might be more compelling to readers than "Dask's data access is now more internally consistent".

Reply:
After reading through the blogpost I'm going to double-down on this point.  Obviously, you should write what you want, but my sense is that "Use Dask's IO System Without Dask" is likely a really attractive message to folks.  I recommend re-orienting the article around this message (but again, that's just my feeling here, please don't let me sway you too much from what you want to write).